src/App.tsx

```diff
@@ -10,33 +12,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setTab] = useState(tabs[0]);
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```
The name of the function must be set according to the noun variable